config/routes.rb

```diff
@@ -6,10 +6,7 @@
       resources :groups do
         post 'add_members', on: :member
         post 'remove_members', on: :member
-        resources :expenses do
-          post 'add_payer', on: :member
-          post 'remove_payer', on: :member
-        end
+        resources :expenses
```

Comment:
Adding a resource will implicitly create unnecessary endpoints for update, delete etc. Make sure you are whitelisting only the required actions using :only / :except filters